typegate/prod/src/logger.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+use colored::Colorize;
+use log::Level;
+use std::io::Write;
+
+pub fn init() {
+    if std::env::var("RUST_LOG").is_err() {
+        std::env::set_var("RUST_LOG", "info,swc_ecma_codegen=off,tracing::span=off");
```

Comment:
is there swc_ecma_codegen inside the typegate?

Reply:
That's the library deno itself uses for transpilation and it emits a lot of `ERROR` and `INFO` logs. I did some cursory investigation and it wasn't clear what the issue was. `tracing::span` on the other hand might be a incompatibility  between a `tracing` based dep and `env_logger`.